src/Symfony/Component/Notifier/Bridge/LightSms/LightSmsTransport.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LightSms;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class LightSmsTransport extends AbstractTransport
+{
+    protected const HOST = 'www.lightsms.com';
+
+    private $login;
+    private $password;
+    private $from;
+
+    private const ERROR_CODES = [
+        000  => 'Service unavailable',
```

Comment:
`000` will not work, is this really the code which comes from the API?

cc @jderusse

Reply:
Checked it now with validation without WWW. API return back.

```LightSmsTransport.php on line 122:
array:1 [▼
  "" => array:1 [▼
    "error" => 31
  ]
]

Reply:
They have two errors without www and main errors from api, e.g. balance :)

Reply:
But you cannot have `000` as key IMHO

Reply:
In reality never had 000 :))) in this service :)

Reply:
Then it should be removed from the constant

Reply:
@OskarStark removed for now :)